generator/generator_tests/osm_type_test.cpp

```diff
@@ -473,6 +473,21 @@ UNIT_CLASS_TEST(TestWithClassificator, OsmType_Layer)
     TEST_EQUAL(params.layer, 1, ());
   }
 
+  /* TODO: add an explicit layer=1 for ANY bridge/tunnel value except "no"
+  {
+    Tags const tags = {
+      { "highway", "secondary" },
+      { "bridge", "positive_value" },
+    };
+
+    auto const params = GetFeatureBuilderParams(tags);
+
+    TEST_EQUAL(params.m_types.size(), 1, (params));
+    TEST(params.IsTypeExist(GetType({"highway", "secondary", "bridge"})), ());
+    TEST_EQUAL(params.layer, 1, ());
+  }
+  */
```

Comment:
I'd prefer to trust OSM here. There are a lot of multi-level junctions, so it's not safe just to assign 1.

Reply:
not sure what you mean...
atm we assign `layer=1` automatically for `bridge=yes` (see the test just above)
why not assign it to e.g. `bridge=viaduct` also?

Reply:
hm, I missed that